docs/current_docs/adopting.mdx

```diff
@@ -4,13 +4,121 @@ slug: /adopting
 
 # Adopting Dagger
 
-You’ve done the [quickstart](quickstart/index.mdx), and now you’re probably asking yourself, "What should I do next?" 
+Once you have [completed the quickstart](quickstart), and learned the basic concepts, it's time to take the next step: adopting Dagger in your project - what we call *daggerizing*. But how does one Daggerize a project, exactly?
 
-We get it, looking at that many lines of YAML can be intimidating. 
-Since each pipeline is unique in tooling and functionally, our general answer is to "start simple". Pick a part of your pipeline to Daggerize first, and then continue to Daggerize from there.
+Every project is different, and every pipeline is a unique snowflake (usually that's not a good thing). So we can't give you an exact step-by-step guide of how to daggerize your project. But we can share the collective experience of the Dagger community, distilled into a simple framework.
 
-Not sure which part of your pipeline to Daggerize first? Reach out to us on [Discord](https://discord.gg/dagger-io) via our [help forum](https://discord.com/channels/707636530424053791/1030538312508776540). Our team and your fellow Daggernauts will be happy to provide more specific guidance on your use case there! 
+## Phase 1: research
```

Comment:
Right before this section might be a good place to put a simple diagram showing the three stages.

Reply:
Good idea. Let's leave it for a follow-up PR if you don't mind?